tarodb/assets_store_test.go

```diff
@@ -115,6 +121,8 @@ func defaultAssetGenOpts(t *testing.T) *assetGenOptions {
 			},
 		}),
 	}
+
+	return a
```

Comment:
What's the significance of this? Or just left over from a different refactor?

Reply:
Left over from when I needed to examine an internal field, removed.